babel.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       }
     ]
   ],
-  plugins: ['@babel/plugin-proposal-class-properties'],
+  plugins: ['@babel/plugin-proposal-class-properties', 'emotion'],
```

Comment:
@horacioh seems like this should be removed based on what you said about styled components

Reply:
uops! my bad...

Reply:
thanks for the review!